src/admin/mod.rs

```diff
@@ -180,10 +180,11 @@ impl Server {
                                 "/config_dump" => Ok::<_, hyper::Error>(
                                     handle_config_dump(workload_info, req).await,
                                 ),
+                                "/help" => Ok::<_, hyper::Error>(handle_help(req).await),
```

Comment:
I agree with @hzxuzhonghu, this should be served as a "menu" when visting `/`.

Reply:
> 

make sense, thanks you @hzxuzhonghu @PiotrSikora, will change it

Reply:
@hzxuzhonghu @PiotrSikora 
I updated the code, alos put some information and logs in the PR comments. please have a look. Thank you.